.github/workflows/semantic-release.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     if: needs.release.outputs.new_release_published == 'true'
     
     strategy:
-      max-parallel: 1
+      max-parallel: 3
```

Comment:
I set it to `1` because it takes too long and I suspect in parallel it failed more.

Reply:
Its really slow when it is sequential. Theoretically there shouldn't be any relation between them so the parallelism should not affect success rate. Lets see